src/Components/Artwork/GridItem.tsx

```diff
@@ -118,7 +118,7 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
           showHoverDetails={showHoverDetails}
           disableRouterLinking={disableRouterLinking}
           to={to}
-          enableSaveButtonForLists
+          enableSaveButtonForLists={!artwork.isInAuction}
```

Comment:
excludes in auction works from the flow (for artwork grids)

Reply:
should I also remove this from here as well?

Reply:
@dimatretyak

Reply:
yep